backend/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "GDX Agreements Tracker API",
   "main": "src/server.js",
   "scripts": {
-    "start": "NODE_ENV=development nodemon src/server.js",
+    "start": "NODE_ENV=development nodemon --watch src/ src/server.js",
```

Comment:
i found that as i was changing any of the code inside src or its subdirectories (say, adding a new endpoint for example), it would have no effect. i either had to restart the container, or make a dummy change to server.js to trigger nodemon in order to have the new changes brought in. i added the watch, and then i could skip all that; if i save any js, nodemon restarts it nicely, and it was back up in a few seconds with the changes.